app/database_etl/etl_main.py

```diff
@@ -203,21 +220,49 @@ def create_bridge_tables(original_data, multi_select_tables):
 def load_postgres_tables(tables_dict, engine):
     # Load dataframes into postgres tables
     for table_name, table_value in tables_dict.items():
-        table_value.to_sql(table_name,
-                           schema='public',
-                           con=engine,
-                           if_exists='append',
-                           index=False)
+        if table_name == 'research_source':
+            table_value.loc[table_value['estimate_name'] == '0724_DeKalbFulton_CDCCOVID-19ResponseTeam_Female_PopAdj',
```

Comment:
what's the purpose of this lol

Reply:
if this is just test code pls remove

Reply:
OH LMAO just test code to F**CCKK it up haha